check_config_win.py

```diff
@@ -0,0 +1,64 @@
+'''
```

Comment:
Also, let's rename this to `check-config-win.py`, since dashes instead of underscores are slightly more idiomatic for CLI tools.